.github/CODEOWNERS

```diff
@@ -1,2 +1,3 @@
 # Primary owner should be listed first in list of global owners, followed by any secondary owners
-*       @SirSpidey @ocofaigh
+
+* @ocofaigh @daniel-butler-irl
```

Comment:
I think you should add yourself and ask in the channel for a volunteer to be a secondary code owner. (We'll find one if there are no volunteers)

Reply:
+1